python/paddle/tensor/math.py

```diff
@@ -1069,6 +1069,31 @@ def addmm(input, x, y, beta=1.0, alpha=1.0, name=None):
         type="addmm", inputs=inputs, attrs=attrs, outputs={"Out": out})
     return out
 
+def renorm(input, p, dim, max_norm):
```

Comment:
parameter "input" should be "x" like other Python API and "dim" should be "axis"